### ISSUE TITLE
`npm install` is failing but `yarn` works. 

### ISSUE THREAD

**Issue by parasharrajat** [react-native-clipboard/clipboard#126]
There are no details to show. I am not able to install the package into other repo that usages NPM.

```
Thank you for using core-js ( https://github.com/zloirock/core-js ) for polyfilling JavaScript standard library!

The project needs your help! Please consider supporting of core-js on Open Collective or Patreon: 
> https://opencollective.com/core-js 
> https://www.patreon.com/zloirock 

Also, the author of core-js ( https://github.com/zloirock ) is looking for a good job -)


> react-native-macos@0.62.15 postinstall /...............clipboard/node_modules/react-native-macos
> node scripts/postInstall.js


> @react-native-clipboard/clipboard@1.8.4 prepare /............./clipboard
> yarn build

yarn run v1.22.15
$ tsc
node_modules/@types/node/globals.d.ts:47:11 - error TS2300: Duplicate identifier 'AbortController'.

47 interface AbortController {
             ~~~~~~~~~~~~~~~

  node_modules/@types/react-native/globals.d.ts:389:15
    389 declare class AbortController {
                      ~~~~~~~~~~~~~~~
    'AbortController' was also declared here.

node_modules/@types/node/globals.d.ts:60:11 - error TS2300: Duplicate identifier 'AbortSignal'.

60 interface AbortSignal {
             ~~~~~~~~~~~

  node_modules/@types/react-native/globals.d.ts:376:15
    376 declare class AbortSignal {
                      ~~~~~~~~~~~
    'AbortSignal' was also declared here.

node_modules/@types/node/globals.d.ts:67:13 - error TS2300: Duplicate identifier 'AbortController'.

67 declare var AbortController: {
               ~~~~~~~~~~~~~~~

  node_modules/@types/react-native/globals.d.ts:389:15
    389 declare class AbortController {
                      ~~~~~~~~~~~~~~~
    'AbortController' was also declared here.

node_modules/@types/node/globals.d.ts:72:13 - error TS2300: Duplicate identifier 'AbortSignal'.

72 declare var AbortSignal: {
               ~~~~~~~~~~~

  node_modules/@types/react-native/globals.d.ts:376:15
    376 declare class AbortSignal {
                      ~~~~~~~~~~~
    'AbortSignal' was also declared here.

node_modules/@types/react-native/globals.d.ts:376:15 - error TS2300: Duplicate identifier 'AbortSignal'.

376 declare class AbortSignal {
                  ~~~~~~~~~~~

  node_modules/@types/node/globals.d.ts:60:11
    60 interface AbortSignal {
                 ~~~~~~~~~~~
    'AbortSignal' was also declared here.
  node_modules/@types/node/globals.d.ts:72:13
    72 declare var AbortSignal: {
                   ~~~~~~~~~~~
    and here.

node_modules/@types/react-native/globals.d.ts:389:15 - error TS2300: Duplicate identifier 'AbortController'.

389 declare class AbortController {
                  ~~~~~~~~~~~~~~~

  node_modules/@types/node/globals.d.ts:47:11
    47 interface AbortController {
                 ~~~~~~~~~~~~~~~
    'AbortController' was also declared here.
  node_modules/@types/node/globals.d.ts:67:13
    67 declare var AbortController: {
                   ~~~~~~~~~~~~~~~
    and here.


Found 6 errors.
```

**Comment by parasharrajat** [comment]
cc: @Naturalclar

**Comment by Naturalclar** [comment]
@parasharrajat are you using typescript in your project?
the error message looks like 	`@types/node` and `@types/react-native` is conflicting which is causing the error.

you may need to have `"skipLibCheck" : true` in your `tsconfig.json` to bypass the error

**Comment by parasharrajat** [comment]
Thanks for giving the solution. I found that already but I can't add it in our repo. I am not using typescript in my project.
Even if you run npm install in a clone of this repo, this error will cause it to fail. 

It is a major issue for other projects installing this lib and use npm and no typescript.

**Comment by parasharrajat** [comment]
I just checked that the main needs yarn as a dependency otherwise npm install fails. But yarn is not added as a dev dependency.

**Comment by Naturalclar** [comment]
@parasharrajat sorry for the delay! it should be fixed in v1.8.5
